### ISSUE TITLE
False Positives in CI

### ISSUE THREAD

**Issue by andig** [golangci/golangci-lint-action#1316]
### Welcome

- [x] Yes, I'm using a binary release within 2 latest [releases](https://github.com/golangci/golangci-lint/releases). Only such installations are supported.
- [x] Yes, I've searched similar [issues on GitHub](https://github.com/golangci/golangci-lint/issues) and didn't find any.
- [x] Yes, I've read the `typecheck` section of the [FAQ](https://golangci-lint.run/docs/welcome/faq/#why-do-you-have-typecheck-errors).
- [x] Yes, I've tried with the standalone [linter](https://golangci-lint.run/docs/linters/) if available (e.g., gocritic, go vet, etc.).
- [x] I agree to follow this project's [Code of Conduct](https://github.com/golangci/golangci-lint?tab=coc-ov-file)

### How did you install golangci-lint?

Official GitHub Action

### Description of the problem

Since 2.6.0 I'm seing various false positives: https://github.com/evcc-io/evcc/actions. I suspect golangci-lint since the positives include `vet` and `gci`, independent of any code changes. Latest example is https://github.com/evcc-io/evcc/actions/runs/18933135220/job/54053788167 where generated files are linted.

I did not try linters standalone since multiple affected and no code changes. I'm also- which is confusing- not able to replicate the errors locally with 2.6..0.

### Version of golangci-lint

<details>

```console
$ golangci-lint --version
# Paste output here
```

</details>

### Configuration

<details>

```console
# paste configuration file or CLI flags here
```

</details>

### Go environment

<details>

```console
$ go version && go env
# paste output here
```

</details>

### Verbose output of running

<details>

```console
prepare environment
  Checking for go.mod: go.mod
  Cache hit for: golangci-lint.cache-Linux-2913-405228c3a5ed00c4494c94858156e3b1fb7815c3
  Cache Size: ~0 MB (303045 B)
  /usr/bin/tar -xf /home/runner/work/_temp/53e6732d-1b2a-4561-91db-8e21a1d0e6a8/cache.tzst -P -C /home/runner/work/evcc/evcc --use-compress-program unzstd
  Cache restored successfully
  Restored cache for golangci-lint from key 'golangci-lint.cache-Linux-2913-405228c3a5ed00c4494c94858156e3b1fb7815c3' in 470ms
  Finding needed golangci-lint version...
  Requested golangci-lint 'latest', using 'v2.6.0', calculation took 36ms
  Installation mode: binary
  Installing golangci-lint binary v2.6.0...
```

</details>

### A minimal reproducible example or link to a public repository

<details>

```go
// add your code here
```

</details>

### Validation

- [x] Yes, I've included all information above (version, config, etc.).

### Supporter

- [ ] I am a sponsor/backer through [GitHub](https://github.com/sponsors/golangci/) or [OpenCollective](https://opencollective.com/golangci-lint)

**Comment by ldez** [comment]
I moved the issue into the GitHub action repository.

I added a new option (`debug`) to the action in v9.1.0.

https://github.com/golangci/golangci-lint-action/blob/e7fa5ac41e1cf5b7d48e45e42232ce7ada589601/action.yml#L61-L67

```yml
uses: golangci/golangci-lint-action@v9
with:
  debug: "clean"
```

This will forcefully clean the golangci-lint cache.

**Comment by andig** [comment]
Thank you. Let me close here- GH never gave an insightful reply.
